tortoise/ninja_tortoise.go

```diff
@@ -502,7 +503,7 @@ func (ni *ninjaTortoise) handleIncomingLayer(newlyr *types.Layer) { //i most rec
 			if Window > newlyr.Index() {
 				windowStart = 0
 			} else {
-				windowStart = newlyr.Index() - Window + 1
+				windowStart = Max(ni.pBase.Layer()+1, newlyr.Index()-Window+1)
```

Comment:
Is it possible that Pbase will be bellow the window?

Reply:
it is but it shouldn't happen that means that pbase is stuck and we need to do a roleback/selfhealing/somthing